internal/config/plugin.go

```diff
@@ -42,20 +43,37 @@ func NewPlugins() Plugins {
 
 // Load K9s plugins.
 func (p Plugins) Load() error {
-	return p.LoadPlugins(K9sPlugins)
+	return p.LoadPlugins(K9sPlugins, K9sPluginDir)
 }
 
 // LoadPlugins loads plugins from a given file.
-func (p Plugins) LoadPlugins(path string) error {
+func (p Plugins) LoadPlugins(path string, pluginDir string) error {
 	f, err := os.ReadFile(path)
-	if err != nil {
-		return err
+	var pp Plugins
+	if err == nil {
+		if err := yaml.Unmarshal(f, &pp); err != nil {
+			return err
+		}
 	}
 
-	var pp Plugins
-	if err := yaml.Unmarshal(f, &pp); err != nil {
-		return err
+	pluginFiles, err := os.ReadDir(pluginDir)
+	if err == nil {
+		for _, pluginName := range pluginFiles {
```

Comment:
```suggestion
		for _, file := range pluginFiles {
```

Because you just read a directory and it can have multiple non-plugin files :)

Reply:
Changed 👍️